src/PrestaShopBundle/Resources/config/routing/admin/sell/catalog/product_v2/product.yml

```diff
@@ -1,3 +1,37 @@
+admin_products_v2_index:
```

Comment:
We have `v2` route because there's `v1` routes I guess?

We'll have to think how to perform the switch 🤔  maybe we'll redirect / remap all v1 routes to v2 ?

Reply:
Guys working on product page started naming them `admin_products_v2_create` so I just used the same format. 
I guess it's different for product list because it has to be fully migrated for 1.7.8 and old one will be gone, so I might need to name routes differently. 

Also route admin_products_v1_index  doesn't exist.